smoke/IotEdgeQuickstart/details/Details.cs

```diff
@@ -334,6 +339,64 @@ protected async Task VerifyDataOnIoTHub(string moduleId)
             await eventHubClient.CloseAsync();
         }
 
+        protected async Task VerifyTwinAsync()
+        {
+            if (this.TwinTestFileName.HasValue)
+            {
+                string twinTestJson = JObject.Parse(File.ReadAllText(this.TwinTestFileName.GetOrElse(string.Empty))).ToString();
+
+                var twinTest = JsonConvert.DeserializeObject<TwinTestConfiguration>(twinTestJson);
+
+                Twin currentTwin = await this.context.RegistryManager.GetTwinAsync(this.context.Device.Id, twinTest.ModuleId);
+
+                if (twinTest.TwinTest.ContainsKey("desired") && twinTest.TwinTest["desired"].Count > 0)
+                {
+                    StringBuilder twinpatch = new StringBuilder(@"{ properties: { desired: { ");
+                    foreach (var desiredProperty in twinTest.TwinTest["desired"])
+                    {
+                        twinpatch.Append($@"{desiredProperty.Key} : {desiredProperty.Value},");
```

Comment:
is double quote required for json values?

Reply:
Not for this patch. IT worked e2e.

Reply:
But, nevermind. :) Removed this code.